### PR TITLE
Alcail/a11y bugs on ParticipantList being inaccessible through keyboard

### PR DESCRIPTION
**Pull request by alcail**
# What
made ParticipantItem focusable
changed 'More Options' to be always displayed if there are some items there, so behaviour is easier for accessibility purpose
updated styles
image
image

# Why
ParticipantList and thus ParticipantItems were inaccessible through keyboard navigation, so user could not read the list of participants and thus access participant info or options
bug https://skype.visualstudio.com/SPOOL/_workitems/edit/2562125

# How Tested
ran storybook and calling sample locally

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-components/src/components/ParticipantItem.tsx [hunk 1]
```diff
@@ -77,14 +73,15 @@ export interface ParticipantItemProps {
   strings?: Partial<ParticipantItemStrings>;
 }
 
+const iconsContainerDefaultWidth = '5rem';
```
Comment (alkwa-msft) [inline packages/react-components/src/components/ParticipantItem.tsx:76]: can we not add this as a style in iconContainerStyle?

Comment (alcail) [reply packages/react-components/src/components/ParticipantItem.tsx:76]: sure :)

File: packages/react-components/src/components/styles/ParticipantItem.styles.ts [hunk 2]
```diff
@@ -14,17 +14,17 @@ export const participantItemContainerStyle = mergeStyles({
...
   alignItems: 'center',
-  height: '100%',
-  overflow: 'hidden'
+  height: '100%'
 });
 
-export const iconStyles = mergeStyles({ display: 'flex', margin: 'auto' });
+export const iconStyles = mergeStyles({
+  display: 'flex',
+  lineHeight: 0,
```
Comment (JamesBurnside) [inline packages/react-components/src/components/styles/ParticipantItem.styles.ts:28]: what does line height 0 do here? put a quick comment here to ensure this isn't removed in the future and the bug re-introduced
```suggestion
  lineHeight: 0, // ensure the icon ...
```

**Review by JamesBurnside — APPROVED** [review]

File: packages/react-components/src/components/styles/ParticipantItem.styles.ts (hunk 2 above)
Comment (alcail) [reply packages/react-components/src/components/styles/ParticipantItem.styles.ts:29]: with line-height at 0
image

without line-height at 0, the "More" icon is slightly above center line
image

File: packages/react-components/src/components/styles/ParticipantItem.styles.ts [hunk 3]
```diff
@@ -14,17 +14,18 @@ export const participantItemContainerStyle = mergeStyles({
 });
 
 export const menuButtonContainerStyle = mergeStyles({
-  root: {
-    '&:hover': { background: 'none' },
-    '&:active': { background: 'none' }
-  }
+  paddingTop: '0.25rem'
 });
```
Comment (mgamis-msft) [inline packages/react-components/src/components/styles/ParticipantItem.styles.ts:18]: I don't think we use this style anymore so we can delete this. Other than that LGTM. Thanks for awesome refactor of ParticipantItem. Looks much simpler now 👍

**Review by mgamis-msft — APPROVED** [review]

**Pull request merged**